pages/dkp/konvoy/2.1/choose-infrastructure/airgapbundle/configurecluster/index.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: layout.pug
+navigationTitle: Bootstrap Cluster
+title: Bootstrap Cluster
+menuWeight: 10
+excerpt: Configure, set up the control plane for, and create your bootstrap cluster.
+beta: false
+enterprise: false
+---
+This section covers the configuration and creation of a bootstrap cluster and it's control plane.
+
+## Configure the bootstrap cluster
+
+1.   To make the following examples easier to follow, set an environment variable with the cluster name.   
```

Comment:
```suggestion
1.  Set an environment variable with the cluster name. to make the following examples easier to follow.
```

Reply:
This had 3 spaces between 1.   and set - is that correct, or is single space, like the following steps below, correct?